zqd/handlers.go

```diff
@@ -274,7 +311,17 @@ func handlePcapPost(c *Core, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	op, err := ingest.NewPcapOp(ctx, s, req.Path, c.ZeekLauncher)
+	pspace, ok := s.(ingest.PcapSpace)
+	if !ok {
+		respondError(c, w, r, zqe.E(zqe.Invalid, "space does not support pcap import"))
+		return
+	}
+	pstore, ok := s.Storage().(ingest.PcapStore)
+	if !ok {
+		respondError(c, w, r, zqe.E(zqe.Invalid, "storage does not support pcap import"))
+		return
+	}
```

Comment:
From first glance and without digging deeper it feels odd that PcapSpace wouldn't have store that implements PcapStore and therefore render this check unnecessary.

Reply:
Oh I guess you do because a log filespace would pass as a PcapSpace but fail as a PcapStorage.

Reply:
In this PR, I did not try to move all of the pcap specific operations from what was a 'Space' to the filestore, hence this split. It seems the me the only complication to doing that is that we track the location of the pcapfile in the space's configuration file currently, and we probably wouldn't want the store to try to write to that.